Library/Formula/pam-u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
```

Comment:
Why aren't we using an upstream release here? They seem to be available. `commit` should be `revision` in this situation FYI.